cpp/src/parquet/metadata.h

```diff
@@ -171,6 +171,7 @@ class PARQUET_EXPORT ColumnChunkMetaData {
 
   const std::vector<Encoding::type>& encodings() const;
   const std::vector<PageEncodingStats>& encoding_stats() const;
+  bool has_bloom_filter() const;
```

Comment:
```suggestion
  std::optional<int64_t> bloom_filter_offset() const;
```

Reply:
It would be more user-friendly to provide this function, like the `GetColumnIndexLocation` below.

Reply:
Thanks, changed as suggested. PTAL.